packages/javascript/bh-shared-ui/src/views/TierManagement/Details/EntityInfo/EntityObjectInformation.tsx

```diff
@@ -23,7 +23,9 @@ import { EntityInfoContentProps } from './EntityInfoContent';
 
 const EntityObjectInformation: React.FC<EntityInfoContentProps> = ({ properties }) => {
     const formattedObjectFields: EntityField[] = formatObjectInfoFields(properties);
-    const nodeProperties = { ...properties, objectid: properties.objectid || '' };
+    const nodeType = properties.nodeType || '';
+    const objectid = properties.objectid || '';
+    const nodeProperties = { ...properties, nodeType, objectid };
```

Comment:
Not positive that the node type is needed here. But added it anyways. I made the prop optional to support it not being used here.